app/keepers/keepers.go

```diff
@@ -507,6 +516,7 @@ func (appKeepers *AppKeepers) initParamsKeeper(appCodec codec.BinaryCodec, legac
 	paramsKeeper.Subspace(incentivestypes.ModuleName)
 	paramsKeeper.Subspace(lockuptypes.ModuleName)
 	paramsKeeper.Subspace(poolincentivestypes.ModuleName)
+	paramsKeeper.Subspace(protorevtypes.ModuleName).WithKeyTable(protorevtypes.ParamKeyTable())
```

Comment:
QQ: What's the reason protorev is the only module using ParamKeyTable?

Reply:
Its essentially just initializing the subspace and parameters here instead of initializing on genesis. I'll change it so that it conforms to the standard.